### ISSUE TITLE
[A11y] Work Show Page-CPD Parent - High contrast

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2014]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

Minor problem in high contrast mode: the download button is weird.

Normal:

![image](https://user-images.githubusercontent.com/2157974/131363368-ccd69ed8-108e-405d-8593-b9cfd73187b8.png)

High contrast:

![image](https://user-images.githubusercontent.com/2157974/131363500-0bef3f26-6ce8-4dfd-a498-e92be061a18b.png)

### Suggested Fix

Same issue reported elsewhere - the "down arrow" is just lost for some reason because it's generated with some weird border magic in CSS. A real icon would probably make a lot more sense here, even if it were a font icon. Possibly better yet would be the "black down-pointing triangle" in unicode: ▼ - though if used, it should be in a screen-reader-invisible span to avoid screen readers saying "Download black down-pointing triangle".

Here's what that looks like in high contrast:

![image](https://user-images.githubusercontent.com/2157974/131365284-aa8c6c5d-c63a-4422-9675-a81d3d654fe5.png)

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Switch to high-contrast mode and check that the down arrow looks good. Make sure the remediation doesn't interfere with screen readers - the down arrow shouldn't be announced.

**Comment by azle** [comment]
Hahahah, apparently you cannot take a screen snap in high contrast mode!

Updated NuChecker 12/26/24
![IMG_8333](https://github.com/user-attachments/assets/b947c883-805d-4e8c-9a3f-17dc8fb895a3)
